recipes/fastjet/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "fastjet" %}
+{% set version = "3.4.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 86dea05ce7a030db9fe14e5bf1c0bc46351b6e439f9b9ef6b14a55ceefcc7b7a
+  patches:
+    - 0001-Don-t-overwrite-external-CXXFLAGS-and-LDFLAGS.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+    - $RPATH/libfastjet.so.0
+    - $RPATH/libfastjettools.so.0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

```suggestion
    - {{ compiler('cxx') }}
    - gxx  # FIXME: 'g++' executable name required for fastjet-contrib
```

So this caused me a huge amount of confusion, but the errors that I was discussing in https://github.com/conda-forge/staged-recipes/pull/21052#issuecomment-1700626956 seem to be because `fastjet-contrib` is looking for the `g++` executable name specifically. This doesn't make much sense to me as https://github.com/scikit-hep/fastjet/pull/191 from @lgray should have already addressed this and `{{ compiler('cxx') }}` should resolve to `gxx` already given https://github.com/conda-forge/staged-recipes/blob/66fd69027ba5308ee8635f63de6677fe6d90bc04/.ci_support/linux64.yaml#L5-L6, but without having `gxx` in here it fails when I run `build-locally.py`. :(